apps/11gist-blog/script.js

```diff
@@ -2,4 +2,34 @@
 
 $(document).ready(function() {
   // You code here
+  //pull post from foxjo github gists
+  $.ajax('https://api.github.com/users/foxjo/gists', {
+    success: function(posts) {
+      posts.forEach(function(post) {
+          if (post.description.startsWith('#post')) {
+            var strPostDescription = post.description.slice(5);
+            var $post = $('<li><a href="#" data-url="' + post.url + '" data-comments="' + post.comments_url + '">' + strPostDescription + '</a></li>');
+            $('#posts').append($post);
+          };
+        }) //append post content when title link is clicked
+      $('body').on('click', 'a', function(event) {
```

Comment:
Why this format of click handler? I'd rather see this the way we did it in class